bool joint_motion
bool cartesian_motion
float64[] point1
float64[] point2
float64[] pose1
float64[] pose2
float64 joint_velocity
float64 joint_acceleration
float64 linear_velocity
float64 linear_acceleration
---
bool success